src/control/server/ctl_ranks_rpc_test.go

```diff
@@ -858,7 +802,8 @@ func TestServer_CtlSvc_StartRanks(t *testing.T) {
 				tc.engineCount = maxEngines
 			}
 
-			ctx := context.Background()
+			ctx, cancel := context.WithTimeout(context.Background(), time.Second)
```

Comment:
Again, I don't think it should always be 1 second. Better to set the test timeout only when we're testing the function's response to a canceled context.

Reply:
as above, I am waiting for confirmation before making further changes

Reply:
removed the fixed time out, done